index.html

```diff
@@ -243,7 +249,7 @@
 
             <ol>
                 <li>
-					the <a href="https://s3.amazonaws.com/pr-preview/tcole3/wpub/master.html#dfn-url">URL</a> of the overall resource; we will refer to this as the <a>Source</a>.
+					the <a href="#dfn-url">URL</a> of the overall resource; we will refer to this as the <a>Source</a>.
```

Comment:
Minor thing: this should actually point to https://w3c.github.io/wpub/index.html#dfn-url, because there is no local definition of a URL. (The original one was indeed a leftover editorial bug.)